tests/test_exapp.py

```diff
@@ -31,9 +31,9 @@ def hello_command_handler(myarg=None, abc=None):
 class MyCommandsLoader(CLICommandsLoader):
 
     def load_command_table(self, args):
-        with CommandGroup(self, 'hello', '__main__#{}') as g:
+        with CommandGroup(self, 'hello', '{}#{{}}'.format(__name__)) as g:
```

Comment:
`__main__` denotes this file only when this file is directly invoked by `python`.

When invoked from `pytest`, `__main__` is actually `D:\cli\py311\Scripts\pytest.exe\__main__.py`. This can be verified by adding `import __main__; print(__main__)`:

```
> pytest examples\test_exapp.py -s
================================================= test session starts =================================================
platform win32 -- Python 3.11.9, pytest-8.2.2, pluggy-1.5.0
rootdir: D:\cli\knack
plugins: forked-1.6.0, xdist-3.6.1
collecting ... <module '__main__' from 'D:\\cli\\py311\\Scripts\\pytest.exe\\__main__.py'>
```